frontend-implicaction/src/app/admin/jobs/components/pending-job-table/pending-job-table.component.ts

```diff
@@ -0,0 +1,57 @@
+import {Component} from '@angular/core';
+import {BaseWithPaginationComponent} from '../../../../shared/components/base-with-pagination/base-with-pagination.component';
+import {JobPosting} from '../../../../shared/models/job-posting';
+import {ToasterService} from '../../../../core/services/toaster.service';
+import {JobService} from '../../../../job/services/job.service';
+import {ActivatedRoute} from '@angular/router';
+import {finalize, take} from 'rxjs/operators';
+import {Criteria} from '../../../../shared/models/Criteria';
+
+@Component({
+  selector: 'app-pending-job-table',
+  templateUrl: './pending-job-table.component.html',
+  styleUrls: ['./pending-job-table.component.scss']
+})
+export class PendingJobTableComponent extends BaseWithPaginationComponent<JobPosting, Criteria> {
+
+  isLoading = true;
+  // Pagination et filtres
+  rowsPerPage = this.pageable.rowsPerPages[0];
+
+  constructor(
+    private toastService: ToasterService,
+    private jobsService: JobService,
+    protected route: ActivatedRoute
+  ) {
+    super(route);
+  }
+
+  validateJob(job: JobPosting): void {
+    this.jobsService
+      .validateJob(job)
+      .subscribe(
+        () => this.paginate({first: this.pageable.first, rows: this.pageable.rows}),
```

Comment:
normalement tu peux lui passer direct pageable : 
```suggestion
        () => this.paginate(this.pageable),
```